### PR TITLE
mrc-402 Added onetime token table

### PR DESCRIPTION
**Pull request by EmmaLRussell**
Adding DB table for storing onetime tokens used in request password reset links

### CODE REVIEW

File: migrations/sql/V2019.07.29.1316__InitialSchema.sql [hunk 1]
```diff
@@ -7,3 +7,9 @@ CREATE TABLE users
   serializedprofile varchar(10000),
   PRIMARY KEY ("id")
 );
+
+CREATE TABLE onetime_token
+(
+  id serial primary key,
```
Comment (hillalex) [inline migrations/sql/V2019.07.29.1316__InitialSchema.sql]: Why have a numerical id? The tokens are presumably unique?

Comment (EmmaLRussell) [reply migrations/sql/V2019.07.29.1316__InitialSchema.sql]: This is how it is in Montagu, not sure why!

Comment (EmmaLRussell) [reply migrations/sql/V2019.07.29.1316__InitialSchema.sql]: Did tokens potentially get reused in Montagu? Agree shouldn't be any need for us to have this atm, will take out.

Comment (hillalex) [reply migrations/sql/V2019.07.29.1316__InitialSchema.sql]: I just dug around and found this explanation as to why we used a numerical id in Montagu! https://vimc.myjetbrains.com/youtrack/issue/VIMC-2087

I _think_ in this case we should be alright as we won't be encoding roles/permissions in the token

**Review by hillalex — APPROVED** [review]

File: migrations/sql/V2019.07.29.1316__InitialSchema.sql (hunk 1 above)
Comment (EmmaLRussell) [reply migrations/sql/V2019.07.29.1316__InitialSchema.sql]: Ah ok, makes sense. Seems to work ok from my testing so far...

**Pull request merged**